bot/cogs/tags.py

```diff
@@ -139,6 +141,26 @@ def _get_tags_via_content(self, check: Callable[[Iterable], bool], keywords: str
                 max_lines=15
             )
 
+    async def send_embed_with_trashcan(self, ctx: Context, embed: Embed) -> None:
```

Comment:
A function for this already exists [here](https://github.com/python-discord/bot/blob/master/bot/utils/messages.py#L16).